pkg/network/cache/multi.go

```diff
@@ -171,7 +171,7 @@ func (x *multiClient) iterateClients(ctx context.Context, f func(clientcore.Clie
 }
 
 func (x *multiClient) ReportError(err error) {
-	if errors.Is(err, errRecentlyFailed) {
+	if errors.Is(err, errRecentlyFailed) || errors.Is(err, context.Canceled) {
```

Comment:
Supposedly, this is _very_ important for failover scenario: otherwise we have a created connection is use full `stream_timeout` ~ 10s on each operation.
Can we first see if increasing `opened_cache_capacity` by itself solves the problem?

Reply:
in fact, that is not the reason of the node freezing so i can drop that too. but i cant get why you think that a canceled context is a good reason for client sleep

Reply:
Right now this is how other node's failure manifests itself. Any non-status error (minus split info-related) can signal about potential problems with the connection.

Reply:
dropped that commit. lets discuss `context.*` errors in another issue